App Templates: add info about theme switching (#7029)
DevExpress/devextreme-documentation: concepts/50 React Components/50 Application Template/40 Configure Themes/10 Switch the Theme.md

[note]

> To learn more information about switching themes, refer to the following help topic: [Switch Between Themes at Runtime](/concepts/60%20Themes%20and%20Styles/05%20Predefined%20Themes/60%20Switch%20Between%20Themes%20at%20Runtime '/Documentation/Guide/Themes_and_Styles/Predefined_Themes/#Switch_Between_Themes_at_Runtime'). You can also take a look at this GitHub Example:

Reviewer: ```suggestion
For more information about switching themes, refer to the following help topic: [Switch Between Themes at Runtime](/concepts/60%20Themes%20and%20Styles/05%20Predefined%20Themes/60%20Switch%20Between%20Themes%20at%20Runtime '/Documentation/Guide/Themes_and_Styles/Predefined_Themes/#Switch_Between_Themes_at_Runtime'). You can also take a look at this GitHub Example:
```

You cannot 'learn' more information

#include btn-open-github with {
    href: "https://github.com/DevExpress-Examples/devextreme-project-template-theme-selector"
}

[/note]

> In the DevExtreme React Template, you can switch between themes. Inspect the theme switcher logic in `src\theme.tsx`.

Reviewer: ```suggestion
You can switch between themes in the DevExtreme React Template. `src\theme.tsx` contains the theme switcher logic.
```

> The template uses a main theme for view content and an additional theme ([color swatch](/concepts/60%20Themes%20and%20Styles/05%20Predefined%20Themes/55%20Color%20Swatches.md '/Documentation/Guide/Themes_and_Styles/Predefined_Themes/#Color_Swatches')) for the navigation menu. The corresponding .JSON files are the following:

Reviewer: ```suggestion
The template uses a main theme for view content and an additional theme ([color swatch](/concepts/60%20Themes%20and%20Styles/05%20Predefined%20Themes/55%20Color%20Swatches.md '/Documentation/Guide/Themes_and_Styles/Predefined_Themes/#Color_Swatches')) for the navigation menu. The corresponding .JSON files are shown below.
```

<table class="dx-table">
    <tr>
        <th>Theme</th>
        <th>Base file</th>
        <th>Additional file</th>
    </tr>
    <tr>
        <td><b>Light</b></td>
        <td><code>src\themes\metadata.base.json</code></td>
        <td><code>src\themes\metadata.additional.json</code></td>
    </tr>
    <tr>
        <td><b>Dark</b></td>
        <td><code>src\themes\metadata.base.dark.json</code></td>
        <td><code>src\themes\metadata.additional.dark.json</code></td>
    </tr>
</table>

> The default theme is `fluent.blue.light`. To switch to another theme, open the .JSON file and assign a theme name to the `baseTheme` field:

Reviewer: ```suggestion
The default theme is `fluent.blue.light`. To switch to another theme, open the .JSON file and assign the theme name to the `baseTheme` field:
```
